docs/conf.py

```diff
@@ -91,7 +91,7 @@ def _config(key):
 sys.path.append(os.path.abspath("./_ext"))
 
 extensions = [
-    'sphinxcontrib.httpdomain', 'sphinx.ext.extlinks', 'sphinx.ext.ifconfig',
+    'envoy.docs.sphinx_runner.ext.httpdomain', 'sphinx.ext.extlinks', 'sphinx.ext.ifconfig',
```

Comment:
What's going on here? The rest looks fine.

Reply:
the httpdomain extension (still) doesnt work with multi-core builds (https://github.com/sphinx-contrib/httpdomain/issues/45)

im going to submit a PR to fix upstream, but in the meantime the fix is here https://github.com/envoyproxy/pytooling/blob/main/envoy.docs.sphinx_runner/envoy/docs/sphinx_runner/ext/httpdomain.py